adapters/repos/db/index_queue.go

```diff
@@ -990,7 +1006,7 @@ func (q *vectorQueue) Iterate(allowlist helpers.AllowList, fn func(objects []vec
 	return nil
 }
 
-func (q *vectorQueue) Delete(ids []uint64) {
+func (q *vectorQueue) Delete(ids ...uint64) {
```

Comment:
just wondering why a slice is not preferable in this case

Reply:
Because in some case we want to use it with one id, and in some other with more than one (in a different branch), it's simply more flexible with a variadic. And more consistent with the other Delete methods